Sessions/session1.html

```diff
@@ -0,0 +1,209 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Home</title>
+        <link rel="stylesheet" href="session2.css">
+    </head>
+    
+    <body>
+        <div class="first_section">
+            <header class="header">
+                <div class="logo">
+                    <img src="Images/logo.png" alt="Logo Title">
+                </div>
+
+                <div class="menu">
+                    <ul>
+                        <li><a href="#">Home</a></li>
+                        <li><a href="#">Features</a></li>
+                        <li><a href="#">Reviews</a></li>
+                        <li><a href="#">Download</a></li>
+                    </ul>
+                </div>
+
+                <div class="icons">
+                    <img src="Images/facebookLogo.png" alt="facebook">
+                    <img src="Images/instagramIcon.png" alt="instagram">
+                    <img src="Images/googleLogo.png" alt="google+">
+                </div>
+            </header>
+
+
+            <div class="cover">
+                <div id="heading">
+                    <h1>Get the most fun weather app</h1>
+                    <a href="#">Download</a>
+                    <a href="#">Features</a>
+                </div>
+
+                <div id="phone_image">
+                    <img src="Images/phone1.png" alt="Phone">
+                </div>
+            </div>
+        </div>
+        
+        
+        <div class="features_section">

```

Comment:
HTML element's id and class name should be relevant, plurals and `ings` should not be used.

https://en.bem.info/methodology/html/

Reply:
done